cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -502,8 +506,12 @@
         query_out = None
 
         _, self.config_data_server = self.set_config()
-        
-        job = job_factory(self.par_dic['instrument_name'],
+        instrument_name = time_request = ""
+        if 'instrument_name' in self.par_dic:
```

Comment:
you can use things like:

```python
instrument_name = self.par_dic.get('instrument_name', '')
```

which is reasonably readable and a lot shorter. 
Or you intentionally avoided it?

Reply:
Thanks I will use this approach, much better indeed